tests/neg/named-tuple-ops-copy.scala

```diff
@@ -0,0 +1,16 @@
+//> using options -experimental
+
+trait Mod
+
+given Conversion[String, Mod] = _ => new Mod {}
+
+type Foo = (name: String, mod: Mod)
+case class Foo0(name: String, mod: Mod)
+
+@main def Test =
+  val foo: Foo = (name = "foo", mod = "some_mod")
+  val foo_updated: Foo = foo.copyFrom((mod = "bar")) // error, stays as String
```

Comment:
here is a demo of the limitation of no target typing, which I guess could be mitigated with a dedicated feature, or a secondary library solution based on lenses